subsys/net/lib/azure_iot_hub/Kconfig

```
@@ -5,18 +5,58 @@
 #
 
 module = AZURE_IOT_HUB
-module-prompt = "Azure IoT Hub [EXPERIMENTAL]"
+module-prompt = "Azure IoT Hub"
 orsource "../Kconfig.cloud_$(CLOUD_SERVICE_SELECTOR)"
 
 menuconfig $(module)
 	bool
-	prompt "Azure IoT Hub [EXPERIMENTAL]" if !CLOUD_SERVICE_MUTUAL_EXCLUSIVE
+	prompt "Azure IoT Hub" if !CLOUD_SERVICE_MUTUAL_EXCLUSIVE
+	select AZURE_SDK
+	select NET_SOCKETS_POSIX_NAMES if !POSIX_API
 	select MQTT_LIB
 	select MQTT_LIB_TLS
 	select EXPERIMENTAL
 
 if AZURE_IOT_HUB
 
+config AZURE_IOT_HUB_DEVICE_ID
+	string "Device ID"
+	help
+	  Device ID to be used when connecting to IoT hub and optionally DPS.
+	  Providing a device ID can also be done at run-time using the appropriate APIs.
+
+config AZURE_IOT_HUB_HOSTNAME
+	string "Azure IoT Hub hostname"
+	help
+	  Hostname to Azure IoT Hub instance.
+
+config AZURE_IOT_HUB_PORT
+	int "Azure IoT Hub port"
+	default 8883
+	help
+	  Port number to connect to on Azure IoT Hub instance.
+
+config AZURE_IOT_HUB_SEC_TAG
+	int "TLS sec tag"
+	help
+	  Security tag where TLS credentials are stored.
+
+config AZURE_IOT_HUB_AUTO_DEVICE_TWIN_REQUEST
+	bool "Request device twin automatically when connected"
+	default y
+	help
+	  When enabled, the library will automatically request the device twin upon connection to
+	  the IoT Hub.
+
+config AZURE_IOT_HUB_DPS
+	bool "Use Device Provisioning Service"
```

Please remove `use` as per guide lines:                                                                                                                                                                                                                                                                                                                                   
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings                                                                                                                                                                                                                                                                                              
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.
```suggestion
	bool "Device Provisioning Service"
```